### PR TITLE
check md5sum of comdb2 executable for each test

### PR DESCRIPTION
**Pull request by adizaimi**
Make sure that we run the same executable for all the tests. Changing the comdb2 executable in the src directory will cause the next testcase of the already-running-test to fail.

### CODE REVIEW

File: tests/setup [hunk 1]
```diff
@@ -214,10 +214,22 @@ fi
 #    exit 1
 #fi
 
+compare_md5_sum()
+{
+    loc_comdb2md5sum=$1
```
Comment (nirbhayc) [inline tests/setup]: I think you can skip loc_comdb2md5sum and use $1 directly.

File: tests/setup [hunk 2]
```diff
@@ -214,10 +214,22 @@ fi
...
+    if [ "$loc_comdb2md5sum" != "$COMDB2MD5SUM" ]; then
+        echo "MD5SUM $COMDB2MD5SUM does not match current $loc_comdb2md5sum"
```
Comment (nirbhayc) [inline tests/setup]: "comdb2 md5 checksum mismatch, expected: $COMDB2MD5SUM got: $loc_comdb2md5sum" ?

File: tests/Makefile [hunk 3]
```diff
@@ -12,6 +12,7 @@ export SRCHOME
 export TESTID:=$(shell tools/get_random.sh)
 CURRDIR:=$(shell pwd)
 export TESTDIR:=$(CURRDIR)/test_$(TESTID)
+export COMDB2MD5SUM:=$(shell md5sum ${SRCHOME}/comdb2 | cut -d ' ' -f1)
```
Comment (nirbhayc) [inline tests/Makefile]: Please add a comment for this.

**Comment by adizaimi** [comment]
thanks Nirbhay.

**Pull request merged**